src/main/java/io/github/seggan/sfcalc/Calculator.java

```diff
@@ -43,11 +43,19 @@ public Calculator(SFCalc plugin) {
     public void printResults(CommandSender sender, String command, SlimefunItem item, long amount) {
         Map<String, Long> results = calculate(item);
 
-        sender.sendMessage(String.format(plugin.headerString, ChatColor.stripColor(ItemUtils.getItemName(item.getItem()))));
+        String header;
+        String name = ChatColor.stripColor(ItemUtils.getItemName(item.getItem()));
+        if (amount == 1) {
+            header = String.format(plugin.headerString, name);
+        } else {
+            header = String.format(plugin.headerAmountString, amount, name);
+        }
```

Comment:
I said on discord

Reply:
use my built in format here

Reply:
but it workssss

Reply:
even if u switch around the %s and %d in the header string?

Reply:
idk ill try later